### PR TITLE
Feat: Add Resoto

### PR DESCRIPTION
**Pull request by aquamatthias**
## BACKGROUND

Add Resoto as 1-click application to DigitalOcean marketplace.

-----------------------------------------------------------------------

## Changes
Genesis. Initial version.

-----------------------------------------------------------------------

## Checklist
- [x] review the [contributing doc](https://github.com/digitalocean/marketplace-kubernetes/blob/master/CONTRIBUTING.md) with steps for both adding or updating your application (if applicable)
------------------------------------------------------------------------

Reviewer: @marketplace-eng

### CODE REVIEW

**Review by lloesche — APPROVED** [review]
Nice!

File: stacks/resoto/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+helm repo add someengineering https://helm.some.engineering
+helm repo update > /dev/null
+
+################################################################################
+# chart
+################################################################################
+STACK="resoto"
+CHART="someengineering/resoto"
+NAMESPACE="resoto"
+values=values.yaml
```
Comment (mbeyt) [inline stacks/resoto/deploy.sh:17]: You'll want this to be the Github hosted version of values.yaml, as below, both here and in the upgrade script:
```
if [ -z "${MP_KUBERNETES}" ]; then
  # use local version of values.yml
  ROOT_DIR=$(git rev-parse --show-toplevel)
  values="$ROOT_DIR/stacks/resoto/values.yml"
else
  # use github hosted master version of values.yml
  values="https://raw.githubusercontent.com/digitalocean/marketplace-kubernetes/master/stacks/resoto/values.yml"
fi
```

**Review by mbeyt — CHANGES_REQUESTED** [review]
This mostly looks good. One small change to both the install and upgrade scripts, and I'll approve it.

File: stacks/resoto/deploy.sh (hunk 1 above)
Comment (aquamatthias) [reply stacks/resoto/deploy.sh]: Thanks @mbeyt. I added the lines into deploy and upgrade.

**Review by mbeyt — APPROVED** [review]

**Pull request merged**